modules/mining-pipeline/jimple-evosuite-test-generator/src/main/kotlin/org/cafejojo/schaapi/miningpipeline/testgenerator/jimpleevosuite/EvosuiteTestGenerator.kt

```diff
@@ -31,18 +31,24 @@ internal class EvoSuiteRunner(
      */
     fun run() = receiveProcessOutput(buildProcess())
 
-    private fun buildProcess() = ProcessBuilder(
-        "java",
-        "-cp", System.getProperty("java.class.path"),
-        "org.evosuite.EvoSuite",
-        "-class", fullyQualifiedClassName,
-        "-base_dir", outputDirectory,
-        "-projectCP", classpath,
-        "-Dno_runtime_dependency=true",
-        "-Dsearch_budget=$generationTimeoutSeconds",
-        "-Dstatistics_backend=NONE",
-        "-Doutput_granularity=TESTCASE"
-    ).start()
+    private fun buildProcess(): Process {
```

Comment:
Maybe you could write this as:
```suggestion
    private fun buildProcess() = ProcessBuilder(
            "java",
            "-cp", System.getProperty("java.class.path"),
            "org.evosuite.EvoSuite",
            "-class", fullyQualifiedClassName,
            "-base_dir", outputDirectory,
            "-projectCP", classpath,
            "-Dno_runtime_dependency=true",
            "-Dsearch_budget=$generationTimeoutSeconds",
            "-Dstatistics_backend=NONE",
            "-Doutput_granularity=TESTCASE"
        ).apply {
            environment()["JAVA_HOME"] = System.getProperty("java.home")
        }.start()
```
Avoid introducing temporary local variables.

Reply:
Good suggestion, fixed!

Reply:
*applied! (no pun intended)